cx_Freeze/samples/sqlite/test_sqlite3.py

```diff
@@ -29,4 +29,6 @@
         f.write('%s\n' % line)
 
 print('dump.sql created')
-con.close()
\ No newline at end of file
+con.close()
+
+input("Press ENTER")
```

Comment:
Why are you adding these lines in each of the samples?

Reply:
To facilitate when testing on windows. In order not to have to open a command window and test it, because if you test directly in the window you are building with build\exe.....\test,exe, sometimes you pass the test, but outside, errors can occur when dll library is missing.

Reply:
Please remove those. They are annoying for those who aren't using your particular configuration. If you want you can always create a batch file that contains a "pause" in it. If you really want them for Windows, though, please add a check to make sure that you are on Windows -- so the rest of us don't have to deal with it. ;-)

Reply:
ok. removed.